client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -113,8 +116,8 @@ module.exports = React.createClass( {
 
 	otherCommentSettings: function() {
 		return (
-			<FormFieldset>
-				<FormLegend>{ this.translate( 'Other comment settings' ) }</FormLegend>
+			<FormFieldset className="has-divider">
```

Comment:
I'm not sure that we need to address it in this PR, but maybe we should consider adding a prop instead of directly applying the `has-divider` class.